src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,48 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.io.IOException;
```

Comment:
In your `src/main/java/duke/command/AddTaskCommand.java`, this import line is above other imports from duke, but here it is below the same set of imports. Perhaps make the import sequence the same?